cypress/e2e/Tests/wordpress.cy.js

```diff
@@ -10,22 +10,26 @@ context("Testing posts scenarios", () => {
   });
   it("creating a post", () => {
     cy.get("#title").click().type("title 1");
-    cy.get("input").contains("Opublikuj").click();
-    cy.visit("http://localhost/wordpress/wp-admin/edit.php");
-    cy.findAllByText("title 1").should("be.visible");
+    cy.get("#publish").click();
+    cy.get("#sample-permalink").click();
+    cy.get('a[href="http://localhost/wordpress/2025/01/30/title-1/"]')

```

Comment:
Samo wskazanie tego linku już wystarczy bo najperw sprawdzasz czy link "blabla.com/title-1" istnieje, a potem sprawdzasz czy w nim jest słowo "title-1". Wystarczy cy.get("blabla/title-1").should('exist');